wsimod/nodes/storage.py

```diff
@@ -552,6 +602,54 @@ def __init__(
     #     self.get_dt_excess()['volume'])) _ = self.tank.push_storage(vqip_, force=True)
     #     return self.extract_vqip(vqip, vqip_)
 
+    def apply_overrides(self, overrides=Dict[str, Any]):
+        """Override parameters.
+
+        Enables a user to override any of the following parameters:
+        timearea.
+
+        Args:
+            overrides (Dict[str, Any]): Dict describing which parameters should
+                be overridden (keys) and new values (values). Defaults to {}.
+        """
+        overwrite_params = [
+            "length",
+            "width",
+            "velocity",
+            "damp",
+            "mrf",
+            "uptake_PNratio",
+            "bulk_density",
+            "denpar_w",
+            "T_wdays",
+            "halfsatINwater",
+            "hsatTP",
+            "limpppar",
+            "prodNpar",
+            "prodPpar",
+            "muptNpar",
+            "muptPpar",
+            "max_temp_lag",
+            "max_phosphorus_lag",
+        ]
+
+        for param in overwrite_params:
+            setattr(self, param, overrides.pop(param, getattr(self, param)))
+
+        if "area" in overrides.keys():
+            print(
+                "ERROR: specifying area is depreciated in overrides \
+		for river, please specify length and width instead"
+            )
```

Comment:
Use exceptions.

Reply:
Added runtime errors - is this what is meant?

Reply:
Exactly. You could also use `ValueError`. There are several built-in exception you can sue. Pick the most appropriate one for each case. https://docs.python.org/3/library/exceptions.html